### ISSUE TITLE
iOS 18.1 Release support

### ISSUE THREAD

**Issue by Lixor18** [straight-tamago/misakaX#355]
When theres will be iOS 18.1 Release support?

The logs:
File copied successfully to C:\Users\Vitaliy\Desktop\MisakaX\data/tmp.plist.
iOS Version: 18.1
iOS Name: iPhone 12
[Restore] C:\Users\Vitaliy\Desktop\MisakaX\data/tmp.plist -> /var/containers/Shared/SystemGroup/systemgroup.com.apple.mobilegestaltcache/Library/Caches/com.apple.MobileGestalt.plist
Traceback (most recent call last):
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 87, in
main()
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 46, in main
handle_exception(e, lockdown)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 79, in handle_exception
raise exception
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 44, in main
restore_file(contents=e_contents, backup_path=backup_path, to_path=to_path, lockdown=lockdown)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\exploit\restore.py", line 94, in restore_file
perform_restore(backup=back, lockdown=lockdown)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\exploit_init_.py", line 12, in perform_restore
mb.restore(backup_dir, system=True, reboot=False, copy=False, source=".")
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\pymobiledevice3\services\mobilebackup2.py", line 140, in restore
dl.dl_loop(progress_callback)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\pymobiledevice3\services\device_link.py", line 61, in dl_loop
raise PyMobileDevice3Exception(f'Device link error: {message[1]}')
pymobiledevice3.exceptions.PyMobileDevice3Exception: Device link error: {'ErrorCode': 205, 'ErrorDescription': 'Domain name cannot contain a slash (MBErrorDomain/205)', 'MessageName': 'Response'}
Downloading... [https://github.com/opa334/TrollStore/releases/latest/download/PersistenceHelper_Embedded]
Installing...
Tips app is not installed. Please try after downloading.
[Restore] C:\Users\Vitaliy\Desktop\MisakaX\data/sparserestore/eligibility.plist -> /var/db/eligibilityd/eligibility.plist
Traceback (most recent call last):
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 87, in
main()
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 46, in main
handle_exception(e, lockdown)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 79, in handle_exception
raise exception
File "C:\Users\Vitaliy\Desktop\MisakaX\data\sparserestore\restore.py", line 44, in main
restore_file(contents=e_contents, backup_path=backup_path, to_path=to_path, lockdown=lockdown)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\exploit\restore.py", line 94, in restore_file
perform_restore(backup=back, lockdown=lockdown)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\exploit_init_.py", line 12, in perform_restore
mb.restore(backup_dir, system=True, reboot=False, copy=False, source=".")
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\pymobiledevice3\services\mobilebackup2.py", line 140, in restore
dl.dl_loop(progress_callback)
File "C:\Users\Vitaliy\Desktop\MisakaX\data\python-3.12.5-embed-amd64\Lib\site-packages\pymobiledevice3\services\device_link.py", line 61, in dl_loop
raise PyMobileDevice3Exception(f'Device link error: {message[1]}')
pymobiledevice3.exceptions.PyMobileDevice3Exception: Device link error: {'ErrorCode': 205, 'ErrorDescription': 'Domain name cannot contain a slash (MBErrorDomain/205)', 'MessageName': 'Response'}

![Знімок екрана (1)](https://github.com/user-attachments/assets/e5646fc0-206d-4ce9-8cd9-abb6e7b839e6)

**Comment by MelvinBraemer** [comment]
maybe never, apple fixed it.

**Comment by stuhljaen** [comment]
i really hope it is going to be fixed :/

**Comment by fourcute** [comment]
Is it possible that 18.1 will be supported soon?

**Comment by oood** [comment]
No, they fixed it from the early betas and the official versions iOS 18.1 and iOS 17.7.1 and later no longer support these vulnerabilities.

The vulnerabilities are assigned CVE numbers: CVE-2024-44258 and CVE-2024-44252. As Apple describes them: Restoring a maliciously crafted backup file may lead to modification of protected system files.

https://support.apple.com/en-us/121563
https://support.apple.com/en-us/121567

**Comment by WilliamAxte** [comment]
18.1 ，My God Lord

**Comment by ShaheerIslam1211** [comment]
Oh Man i just updated to ios 18.1 just to use Apple Intelligence but viola everything is broken here didn't hope to see that coming.

**Comment by Sprick2007** [comment]
Same here
